packages/mason/test/bundles/hooks_bundle.dart

```diff
@@ -5,6 +5,7 @@ import 'package:mason/mason.dart';
 
 final hooksBundle = MasonBundle.fromJson(<String, dynamic>{
   "files": [
+    // cspell:disable-next-line
```

Comment:
I think we should likely avoid modifying the bundles and instead ignore them from cspell entirely since they are generated code.